ts/packages/anchor/src/provider.ts

```diff
@@ -367,32 +377,71 @@ export interface Wallet {
 async function sendAndConfirmRawTransaction(
   connection: Connection,
   rawTransaction: Buffer | Uint8Array,
-  options?: ConfirmOptions
+  options?: ConfirmOptions,
+  blockhash?: BlockhashWithExpiryBlockHeight
 ): Promise<TransactionSignature> {
-  const sendOptions = options && {
-    skipPreflight: options.skipPreflight,
-    preflightCommitment: options.preflightCommitment || options.commitment,
+  const sendOptions: SendOptions = {
+    skipPreflight:
+      options?.skipPreflight !== undefined ? options.skipPreflight : true,
+    preflightCommitment:
+      options?.preflightCommitment || options?.commitment || "confirmed",
```

Comment:
It also changes the default `preflightCommitment` to `confirmed` which can also be a breaking change.